alma/lib/Helpers/ModuleHelper.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * 2018-2024 Alma SAS.
+ *
+ * THE MIT LICENSE
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ * documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+ * to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+ * Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+ * WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+ * IN THE SOFTWARE.
+ *
+ * @author    Alma SAS <contact@getalma.eu>
+ * @copyright 2018-2024 Alma SAS
+ * @license   https://opensource.org/licenses/MIT The MIT License
+ */
+
+namespace Alma\PrestaShop\Helpers;
+
+use Alma\PrestaShop\Proxy\ModuleProxy;
+
+if (!defined('_PS_VERSION_')) {
+    exit;
+}
+
+/**
+ * Class ModuleHelper.
+ */
+class ModuleHelper
+{
+    /**
+     * @var ModuleProxy
+     */
+    protected $moduleProxy;
+
+    public function __construct($moduleProxy = null)
+    {
+        if (!$moduleProxy) {
+            $moduleProxy = new ModuleProxy();
+        }
+        $this->moduleProxy = $moduleProxy;
+    }
+
+    /**
+     * @return array
+     */
+    public function getModuleList()
```

Comment:
Is this method return all installed modules even if they're native, or just those which were installed by the merchant?

Reply:
@webaaz All installed modules even if they're native

Reply:
Do we need native ones? It could be too big. How many modules do you send? What's the length of the Json string?